src/controller/room.controller.ts

```diff
@@ -109,3 +113,64 @@ export const postRoom = async (req: NextApiRequest, res: NextApiResponse) => {
     return;
   }
 };
+
+function runMiddleware(
+  req: NextApiRequest & { [key: string]: any },
+  res: NextApiResponse,
+  fn: (...args: any[]) => void
+): Promise<any> {
+  return new Promise((resolve, reject) => {
+    fn(req, res, (result: any) => {
+      if (result instanceof Error) {
+        return reject(result);
+      }
+      return resolve(result);
+    });
+  });
+}
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+export const postRoomThumbnail = async (
+  req: NextApiRequest & { [key: string]: any },
+  res: NextApiResponse
+) => {
+  try {
+    const multerUpload = multer({
+      storage: multer.diskStorage({
+        filename: function (req, file, callback) {
+          const ext = path.extname(file.originalname);
+          callback(null, "test" + ext);
+        },
+      }),
+      limits: { fileSize: 5 * 1024 * 1024 },
```

Comment:
MAX_IMAGE_BYTE_SIZE 라는 상수가 존재하여 이용하는게 좋아보이네요.
```suggestion
      limits: { fileSize: MAX_IMAGE_BYTE_SIZE },
```